docs/terraform.md

```diff
@@ -10,7 +10,7 @@
 
 | Name | Version |
 |------|---------|
-| <a name="provider_aws"></a> [aws](#provider\_aws) | >= 4.22.0 |
+| <a name="provider_aws"></a> [aws](#provider\_aws) | 5.8.0 |
```

Comment:
@dragosmc this change here is what is causing your README failures. I think this may be an issue with an older version of `terraform-docs`. Maybe try upgrading that and then seeing if this gets the changes when you run `make readme` again?

Reply:
I have experienced this issue in the past, where terraform-docs resolves the current provider version instead of maintaining the version constraint. I have always just reverted those changed lines back, if I'm not touching anything related to providers or dependencies.